python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -1428,6 +1437,13 @@ def set_img_parser(parser: ArgumentParser) -> ArgumentParser:
                  """,
         )
 
+        parser.add_argument(
```

Comment:
`test_set_img_parser` in `test_closed_loop_task` should be updated to test this new config.

Reply:
Actually I think this belongs more with the `default_parser` above and not in the `img_parser` which contains settings specific to the images in the simulation. So this should be moved above and then a line in the test for this should be added to `test_default_parser`.